services/web/server/src/simcore_service_webserver/projects/utils.py

```diff
@@ -93,6 +93,12 @@ def _replace_uuids(node: str | list | dict) -> str | list | dict:
             project_copy["ui"].get("slideshow", {})
         )
 
+        # exclude conversations
```

Comment:
Can you explain why in the comments?

Reply:
Because if we copy a project, we don't want to bring the conversations done on the source to the new one. Or did you mean in the code comments? 😅

Reply:
Done in both.